packages/lesswrong/lib/collections/posts/schema.js

```diff
@@ -57,6 +57,13 @@ const schema = {
       }
     }
   },
+  // Timestamp of last post modification
+  modifiedAt: {
```

Comment:
1. Needs a migration script
2. Would be simpler (no need to manually write a callback) using `denormalizedField` in `lib/modules/utils/schemaUtils.js`

Reply:
Do I need to do a migration script in addition to the denormalized field? (I vaguely recall denormalized fields servering the role of migration scripts but can't remember in detail)

Reply:
Yes. The migration script only needs to call `recomputeDenormalizedValues`, though. (See the `fixDeletedBios` migration for an example.)

(Also worth noting: `modifiedAt` will update when *any* change is made to the post, including moving between sections, approving, etc. That's fine for a sort order, but if this ever gets displayed to users, we might want a less inclusive definition of changes.)

Reply:
I was thinking it was actually generally useful to have a "all kinds of modification" field (and had some background belief this was reasonably standard admin-only-field type, more in the reference class of "createdAt" than "postedAt". I think if we need to display "editedAt" we can probably use the revisions table)